projects/core/src/lib/utils/pipe.ts

```diff
@@ -1,15 +1,13 @@
 import {MaskitoPostprocessor, MaskitoPreprocessor} from '../types';
 
 /**
- * @deprecated Use property `preprocessors` from `MaskitoOptions`
  * @internal
  */
 export function maskitoPipe(
```

Comment:
Should we rename it to just `pipe` now that it is not exported?

Reply:
`maskitoPipe` can be useful for unit testing.

For example:
https://github.com/Tinkoff/taiga-ui/blob/673e1344d728a451a6b75f480d698ffae0f82132/projects/addon-commerce/constants/tests/card-holder-mask.spec.ts#L24